app/models/container_route.rb

```diff
@@ -5,4 +5,6 @@ class ContainerRoute < ActiveRecord::Base
   belongs_to :ext_management_system, :foreign_key => "ems_id"
   belongs_to :container_project
   has_many :labels, -> { where(:section => "labels") }, :class_name => "CustomAttribute", :as => :resource
+  belongs_to :container_service
+  has_many :labels, :class_name => CustomAttribute, :as => :resource, :conditions => {:section => "labels"}
```

Comment:
I already commented about this line. It's unrelated to the patch and it's reintroducing old code. It should be removed.